bots/pr/src/main/java/org/openjdk/skara/bots/pr/IntegrateCommand.java

```diff
@@ -159,8 +193,7 @@ public void handle(PullRequestBot bot, PullRequest pr, CensusInstance censusInst
                 return;
             }
 
-            var botUser = pr.repository().forge().currentUser();
-            var backportLines = pr.comments()
+            var backportLines = allComments
```

Comment:
This is a slight optimization. We already have a list of all comments, so no need to reach out to github to fetch them again. I can't see any need in this situation to refresh either.

Reply:
So there's no need to ever refresh after acquiring the integration lock?

Reply:
Not sure, I don't think the IntegrationLock is relevant for this, as it's not involved in the CheckWorkItem where this comment would potentially be created, but maybe I should still leave this alone as I'm not sure about how the concurrency could play out here.

Investigating this made me discover that there is a duplicate implementation of this whole thing in the SponsorCommand that I need to address.